misc/SW_WA_2021_2024/guilderton_to_geraldton/README.md

```diff
@@ -0,0 +1,24 @@
+# Tsunami modelling for the Perth to Geraldton area in 2023-2024
+
+## Part of the WA Tsunami Inundation Modelling Project with DFES
+
+This folder is for placing high-resolution grids throughout the region from perth to Geraldton.
+The main coastline in this stretch is the midwest, although it includes other areas further south down to Guilderton. The codes are derived from the [Greater Perth Modelling files](../Greater_Perth) with some modifications to the folder structure and the model setup. In particular, the elevation data, breakwalls, initial_stage regions have been updated and improved to reflect the new area. Additionally, several large buildings in Geraldtown have been included in the model elevation. With these changes, more refined inundation maps are expected in the region from Guilderton to Geraldton.
```

Comment:
Here could you please make the link be ../greater_perth_revised2023 (which is more correct)

Reply:
Actually don't worry -- I realised it will be more efficient for me to merge first, then make the fix